site-packages/tasks_ci/formatting.py

```diff
@@ -213,6 +216,101 @@ def enforce_quantum_symbols(
 
     return cursor, fail
 
+
```

Comment:
Is the goal here to attempt what the CI will check for but running it locally?

Reply:
This is the script that is both used locally and as part of CI. The main goal of the tool is to correct as much as it can automatically and print out the things it cannot fix autonomously. Checks include such things as copyright in headers, general formatting of code, `qis|rt` function names and a few other things.